blocks/resizer-image-block/getResizedImageData.test.js

```diff
@@ -520,6 +521,29 @@ describe('get resized image data helper on the server-side', () => {
       );
       expect(dataSearch).toEqual(mockSearchApiDataEmptyPromo);
     });
+
+    it('takes in collection api results data object', () => {
+      getProperties.mockImplementation(() => ({
+        breakpoints: DEFAULT_BREAKPOINTS_ARRAY,
+        aspectRatios: ASPECT_RATIOS,
+        imageWidths: IMAGE_WIDTHS,
+        resizerURL: 'https://fake.cdn.com/resizer',
+        shouldCompressImageParams: true,
+      }));
+
+      const dataSearch = getResizedImageData(
+        mockCollectionApiData,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+      );
+
+      expect(typeof dataSearch.content_elements[0].promo_items.basic.resized_params).toEqual(
+        'object',
```

Comment:
not sure I wildly follow this test case. did this fail before? would it have returned null if it wasn't working?

Reply:
This is a new test case, and wouldn't return before